kalendar-endlos/src/main/java/com/himanshoe/kalendar/endlos/utils/Utils.kt

```diff
@@ -0,0 +1,10 @@
+package com.himanshoe.kalendar.endlos.utils
+
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+fun getMonthNameInDeviceLocale(monthIndex: Int): String {
```

Comment:
Dont create a Util file. Make it an extension to Month

Reply:
Found a better way, using existing extension functions.